src/core/common/Breadcrumbs.js

```diff
@@ -36,30 +39,49 @@ const styles = theme => ({
  * This component returns breadcrumbs block
  *
  * @author Bogdan Shcherban <bsc@piogroup.net>
- * @param {shape} classes
- * @param {shape} resource
  */
-const Breadcrumbs = ({ classes, resource }) => {
-    return (
-        <div className={classes.breadcrumbsBlock}>
-            <Typography>
-                <Link to="/" className={classes.link} color="inherit" aria-label="Home">Home</Link>
-            </Typography>
-            {
-                resource.map((item, key) => {
-                    return (
-                        <div key={key} className={classes.breadcrumbsItem}>
-                            <div className={classes.separator}></div>
-                            {item.isActive
-                                ? <Link to={item.url} className={classes.link} color="inherit" aria-label={item.title}>{item.title}</Link>
-                                : <Typography>{item.title}</Typography>
-                            }
-                        </div>
-                    );
-                })
-            }
-        </div>
-    );
+class Breadcrumbs extends Component {
+
+    goHomePage = () => {
+        this.props.removeUserSearch();
+        window.location.replace('/#/');
```

Comment:
If you want to see the location changes in the Redux actions, use the `push` action from 'react-router-redux` instead. This is optional.